crates/bevy_render/src/lib.rs

```diff
@@ -272,6 +272,10 @@ impl Plugin for RenderPlugin {
                         let instance = wgpu::Instance::new(wgpu::InstanceDescriptor {
                             backends,
                             dx12_shader_compiler: settings.dx12_shader_compiler.clone(),
+                            // TODO: make this configurable
+                            flags: wgpu::InstanceFlags::empty(),
+                            // TODO: make this configurable
+                            gles_minor_version: wgpu::Gles3MinorVersion::Automatic,
```

Comment:
Just noting that I'm fine with this as an initial impl.

Reply:
They're configurable now.